app/src/main/java/com/puzzlebench/clean_marvel_kotlin/presentation/fragments/mvp/FragmentDialogPresenter.kt

```diff
@@ -0,0 +1,27 @@
+package com.puzzlebench.clean_marvel_kotlin.presentation.fragments.mvp
+
+import com.puzzlebench.clean_marvel_kotlin.presentation.fragments.CharacterFragmentDialog
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.schedulers.Schedulers
+
+class FragmentDialogPresenter(val view: FragmentDialogView, val model: FragmentDialogModel): FragmentDialogContracts.Presenter {
+
+    override fun init(characterFragment: CharacterFragmentDialog) {
+        requestGetSingleCharacter(characterFragment)
+    }
+
+    private fun requestGetSingleCharacter(fragment: CharacterFragmentDialog){
+        //view.showLoading(fragment)
```

Comment:
uncomment the line if showing the loader or remove it otherwise

Reply:
done